projects/adc_periodic_reader/test/adc_periodc_reader_test.c

```diff
@@ -0,0 +1,53 @@
+#include "adc.h"
+#include "adc_periodic_reader.h"
+#include "delay.h"
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "log.h"
+#include "misc.h"
+#include "ms_test_helpers.h"
+#include "soft_timer.h"
+#include "status.h"
+#include "test_helpers.h"
+

```

Comment:
complete your test. you also have to test it on a controller board. Not passing a `PLATFORM` argument when you're running the tests, would automatically run it on the controlelr board, make sure you have one connected though...